src/main/kotlin/com/github/sanity/kweb/Kweb.kt

```diff
@@ -43,14 +55,14 @@ class Kweb(val port: Int,
            val debug: Boolean = true,
            val refreshPageOnHotswap : Boolean = false,
            val plugins: List<KWebPlugin> = Collections.emptyList(),
-           val appServerConfigurator: (AppServer) -> Unit = {},
+           val appServerConfigurator: (NettyApplicationHost?) -> Unit = {},
            val onError : ((List<StackTraceElement>, JavaScriptError) -> LogError) = { _, _ ->  true},
            val maxPageBuildTimeMS : Long = 200,
            val buildPage: RootReceiver.() -> Unit
 ) {
     companion object: KLogging()
 
-    private val server = AppServer(AppConfiguration(port = port))
+    private var server: NettyApplicationHost? = null
```

Comment:
Hmm, should this need to be a nullable var given that it is initialized in the `init {}` block?